config/proto/probe.proto

```diff
@@ -39,28 +38,19 @@ message HermesProbeDef {
     CEPH = 2;
   }
 
-  // Probe types currently supported
-  enum ProbeType {
-    PROBE_TYPE_UNSPECIFIED = 0;  // For unspecified probe types
-    GET = 1;  // GET probe is for getting an object (file) from a storage system.
-    PUT = 2;  // PUT probe is for putting an object (file) into a storage system.
-    DELETE = 3;  // DELETE probe is for deleting an object (file) from a storage system.
-  }
-
-  TargetSystem target_system = 3;
-  ProbeType probe_type = 4;
+  optional TargetSystem target_system = 3;
 
-  int32 interval_sec = 5;  // Probing interval in seconds, default = 3600
+  optional int32 interval_sec = 4;  // Probing interval in seconds, default = 3600
```

Comment:
Nit: Move the comments above the fields.

Reply:
Ack.